data/templates/Detector_ServiceFabric.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+		| where Timestamp >= startTime and Timestamp <= endTime
+		YOUR_QUERY
+	";
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.ServiceFabric", resourceTypeName: "clusters")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt, null, "GetQuery")),
```

Comment:
Wont this give a compilation error for GetQuery() method?

Reply:
ohh my bad :( I should have had the parameters for ExecuteClusterQuery, not to GetQuery. Thanks for catching this.